manager/allocator/cnmallocator/networkallocator.go

```diff
@@ -980,3 +999,13 @@ func IsBuiltInDriver(name string) bool {
 	}
 	return false
 }
+
+func setIpamSerialAlloc(opts map[string]string) map[string]string {
+	if opts == nil {
+		opts = make(map[string]string)
+	}
+	if _, ok := opts[ipamapi.AllocSerialPrefix]; !ok {
```

Comment:
What happens if the value in the map is set to `"false"`? Should we rename the function to make clear that it only sets serial allocation if no setting was already in place?

Also, `IPAM` in the funtion name should be `IPAM`.

Reply:
ideally I should have added a setter api in ipam libnetwork library. I missed that train. I can probably come back and make this better.